paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/clone/CopyFileOperator.java

```diff
@@ -139,6 +141,37 @@ private Path pathOfTable(Table table) {
         return new Path(table.options().get(CoreOptions.PATH.key()));
     }
 
+    private void copyManifestFile(Path sourcePath, Path targetPath, CloneFileInfo cloneFileInfo)
+            throws IOException, TableNotExistException {
+        Identifier sourceIdentifier = Identifier.fromString(cloneFileInfo.getSourceIdentifier());
+        FileStoreTable sourceTable = (FileStoreTable) sourceCatalog.getTable(sourceIdentifier);
+        FileStore<?> store = sourceTable.store();
+        ManifestFile manifestFile = store.manifestFileFactory().create();
+
+        List<ManifestEntry> manifestEntries =
+                manifestFile.readWithIOException(sourcePath.getName());
+        List<ManifestEntry> targetManifestEntries = new ArrayList<>(manifestEntries.size());
+
+        // clone job will clone the source path to target warehouse path, so the target external
+        // path is null
+        String newExternalPath = null;
```

Comment:
Variable 'newExternalPath' is never used

Reply:
It's used in line 165, we first declare a variable of `String` type, and then call the `public DataFileMeta copy(String newExternalPath) `function. The reason why there are variables of type `String` is because there are many other `copy` functions in `DataMileMeta`.

Reply:
I means that you can write "manifestEntry.file().copy((String) null))" here.

Reply:
got it